Sources/News.php

```diff
@@ -225,11 +225,11 @@ function ShowXmlFeed()
 	// Allow mods to add extra namespaces and tags to the feed/channel
 	$namespaces = array(
 		'rss' => array(),
-		'rss2' => array('atom' => 'http://www.w3.org/2005/Atom'),
-		'atom' => array('' => 'http://www.w3.org/2005/Atom'),
+		'rss2' => array('atom' => 'https://www.w3.org/2005/Atom'),
+		'atom' => array('' => 'https://www.w3.org/2005/Atom'),
```

Comment:
The official documentation is at https://www.w3.org/TR/xml-names/, but basically, if one changes the URI in the namespace declaration in any way, one is in fact declaring an entirely different namespace. For XML documents that require certain namespaces to be defined (both Atom and RSS2 do), changing the URI means that you are not defining the expected namespace and instead have defined some other, unknown namespace.

Also relevant from that documentation is that the URI does not need to be a working URL. It just needs to be a unique and permanent string that matches the criteria for a URI. What the URI is and whether it would point to any actual document if treated as a URL is beside the point.

With uncommon namespaces (e.g. our own `xmlns:smf="http://www.simplemachines.org"`), it is probable that most XML processors won't already know what it is and will therefore just accept whatever is defined in the XML document. But with common namespaces (e.g. Atom's `xmlns:atom="http://www.w3.org/2005/Atom"`)